schema/data/portal/form_json/electricity_and_heat.json

```diff
@@ -60,43 +136,50 @@
     "electricity": {
       "ui:order": [
         "purchased",
-        "sold",
-        "consumedOnSite",
         "generatedOnSite",
+        "consumedOnSite",
+        "sold",
         "*"
       ],
       "purchased": {
-        "ui:col-md": 4
+        "ui:col-md": 3
```

Comment:
why do we need markup in our schema?

Reply:
This is the `uiSchema`, it defines how things are rendered. The actual json-schema is the `schema` object above

Reply:
It's particularly the tightly-coupled-to-bootstrap bit of the `col-md` class that I'm fussing about. It isn't hugely important in this PR, and it would be nice to decouple css from render config. Would it be hard to get the `uiSchema` to process `"ui:width": "one-quarter"` or something like that?

Reply:
It's actually not coupled to bootstrap, but to our code (which happens to use bootstraps naming convention): https://github.com/bcgov/cas-ciip-portal/blob/02ab1208dfe8241f677ab21814d89c09652a8ef3/app/containers/Forms/FormFieldTemplate.tsx#L38-L64

Reply:
In chatting about this, I think there might be further design work surrounding `uiSchema` in future, but this is good as-is and shouldn't block this merge.